javascript/javascript1/week2/mainweek2.js

```diff
@@ -0,0 +1,109 @@
+/* Flight booking fullname function */
+
+function getFullname(firstname, surname, useFormalName) {
+    if (firstname === "") {
+        return "please enter your firstname ";
+    } else if (surname === "") {
+        return firstname + " !" + " Please enter your surname ";
+    } else if (useFormalName) {
+        return "Lord" + " " + firstname + " " + surname;
+    } else {
+        return firstname + " " + surname;
+    }
+
+}
+const fullname1 = getFullname("Deepthi", "");
+const fullname2 = getFullname("Supriya", "Dukka", true);
+
+console.log(fullname1);
+console.log(fullname2);
+
+/* Event application */
+
+
+function getEventWeekday(whatDayEventHeld) {
+    const weekDays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    let currentDate = new Date();
+    let day = currentDate.getDay();
+    let n = (day + whatDayEventHeld) % 7;
+    return "The event is on: " + weekDays[n];
+}
+
+console.log(getEventWeekday(9));
+
+
+/* Weather wear */
+
+function weatherWear(temperature) {
+
+    if (temperature >= 0 && temperature <= 4) {
+        return "Parka Jakke , Sweater and woolen trouser";
+    } else if (temperature > 4 && temperature <= 8) {
+        return "Jacket , Sweater and jeans";
+    } else if (temperature > 8 && temperature <= 15) {
+        return "Jean Jakke , t-shirt and Jeans";
+    } else {
+        return "shorts and t-shirts";
+    }
+}
+
+
+weatherWear(3);
+
+/*Student manager*/
+let class07Students = [];
+
+function addStudentToClass(studentName) {
+    if (class07Students.length >= 6 && studentName !== "Queen") {
+        alert("Cannot add more students to class 07");
```

Comment:
Good idea using alert! Since we don't have a way of interacting with our javascript through the DOM yet, it's best to stick to `console.log`, however definitely keep this in your back pocket for the future.